docs/common-reference-docs/3-configuring-rules.md

```diff
@@ -63,29 +61,29 @@ Note: * indicates a required field.
 
 To find the token details, please refer to this article: [Find Token Details](./finding-token-details)
 
-**chainId***: Enter the chain ID that your token is on.The chainID is a unique ID assigned to each chain(L1 and L2) - For example 1 is the the chainID for Ethereum Mainnet 137 is the chainID of Polygon Mainnet. You can find the chainIds here: [Chainlist](https://chainlist.org/)
+**chainId***: The chainID is a unique ID assigned to each chain(L1 and L2) - For example `1` is the the chainID for Ethereum Mainnet while `137` is the chainID of Polygon Mainnet. You can find more chainIDs on [Chainlist](https://chainlist.org/).
 
-**type***: Select the token type of your token. For example, for EVM based chains, it can ERC-20, ERC-721, ERC-1155, etc.
+**type***: Select the token type of your token. For example, for EVM based chains, it can be ERC-20, ERC-721, ERC-1155, etc. For non-EVM chains, it can be BEP-20, etc. If you are not sure about the token type, you can use the [Find Token Details](./finding-token-details) article to find the token type.
 
-**contractAddress***: Input the contract address of your token.
+**contractAddress***: The contract address of your token.
 
-**tokenId (optional)**: For NFT token types such as ERC-721 & ERC-1155, you can add specific token IDs.
+**tokenId (optional)**: For NFT token types such as `ERC-721` & `ERC-1155`, you can add specific token IDs.
 
-*Note*: When using the ERC-1155 token type, you must enter the token IDs for all tokens in your collection in the token ID section. Token IDs should be separated by one comma, and no spaces. You can also input a range of token IDs, for example: 30-90.
+>When using the ERC-1155 token type, you must enter the token IDs for all tokens in your collection in the token ID section. Token IDs should be separated by one comma, and no spaces. You can also input a range of token IDs, for example: 30-90.
 
-**minToken (optional)**: Input the minimum amount of tokens that are required to get the role.
+**minToken (optional)**: The minimum amount of tokens that are required for the role to be granted.
 
-*Note*: If you do not enter a minimum amount of tokens, the default is 1.
+>If you do not enter a minimum amount of tokens, the default is 1. The value must allways be a string, e.g "1".
 
-**maxToken (optional)**: Input the maximum amount of tokens that are required to get the role.
+**maxToken (optional)**: The maximum amount of tokens that are required to get the role.
 
-*Note*: If you do not enter a maximum amount of tokens, the default is infinity.
+>If you do not enter a maximum amount of tokens, the default is infinity.
 
-**roleID (optional)**: Role ID is a unique identifier for the rules. You can use this to identify the rules in the API response.
+**roleID (optional)**: The unique identifier for the roles. You can use this to identify the roles in the API response.
```

Comment:
please update `role`(applies only to Discord) to `rule`, since in the whole document and tutorial, we are using `rule` as the terminology. ofc the only thing we can’t change is the API field "roleID"